src/player/update/update_roulade.c

```diff
@@ -30,7 +30,7 @@ static void apply_rect(player_t *player, int current_id)
     player->obj->bigdata.sprite_bigdata.rect = (sfIntRect) {rect[0], rect[1],
         rect[2], rect[3]};
     sfSprite_setOrigin(player->obj->drawable.sprite,
-        (sfVector2f) {rect[2] / 2, rect[3] / 2});
+        (sfVector2f) {rect[2] / 2.0, rect[3] / 2.0});
```

Comment:
Magic value ?

Reply:
2.0 ?
it is just that we have the width and heigh and to have the middle you divide by 2